### PR TITLE
Added a page and link about becoming a member

### PR DESCRIPTION
**Pull request by MandyMeindersma**
Simple page for people wanting to become a member. I feel like it is pretty plain now but maybe we could add #43 to it eventually and then it would have a little more oomf. 

desktop
image

mobile
image

mobile menu
image

tablet
image

### CODE REVIEW

**Comment by MandyMeindersma** [comment]
I don't think I like this. I think I want to merge this and "board" and create a "society" tab (but I don't like the name "society")

#64 is related

**Comment by MandyMeindersma** [comment]
maybe "board + members"

File: components/TheMenu.vue [hunk 1]
```diff
@@ -127,6 +127,10 @@ export default {
           name: "Board",
           href: "/board_recruiting",
         },
+        {
+          name: "Become a Member test",
```
Comment (dboydgit) [inline components/TheMenu.vue:131]: Remove the word "test" here

File: pages/become_a_member.vue [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<template>
+    <div class="max-w-7xl mx-auto spx-6 px-6 lg:px-8">
+        <h1 class="text-2xl font-bold py-4">Become a Member</h1>
+        <p class="py-2">
+            Thank you for your interest in becoming a DES member! Firstly we would like to point out that <strong> you do not need to be a member to use the DES Slack, attend meetups and events, or to join the monthly society meeting and AGM. </strong> Membership is only required if you wish to vote, or use DES equipment and resources (like our YouTube channel or streaming tools.)
+        </p>
+        <p class="py-2">
+            If you're still interested in becoming a member, please complete the form below and then e-transfer $20 to memberships@devedmonton.com. Your membership will be valid for one year from the date that your membership was received. After the transfer is sent and you fill out the form you will be added to a google group which contains all our meetings and AGM's you can attend.
```
Comment (dboydgit) [inline pages/become_a_member.vue:8]: I don't think this wording is in line with the by-laws. Membership is indefinite but can be revoked on non-payment of fees, and are not 1 year terms. @MarkBennett maybe you can chip in here as well. Here's my suggestion:

"If you're still interested in becoming a member, please complete the form below and e-transfer the $20 annual membership fees to memberships@devedmonton.com. On receipt of the completed form and membership fess, your application will be reviewed by the board. On approval of your application you will be added to a google group which contains all our meetings and AGM's you can attend."

**Review by dboydgit — CHANGES_REQUESTED** [review]
@MarkBennett I think this satisfies #66 but can you have a quick look and confirm that the wording works for you.

File: pages/become_a_member.vue (hunk 2 above)
Comment (MarkBennett) [reply pages/become_a_member.vue:8]: [The bylaws talk about fees and status in section 3.06.](https://docs.google.com/document/d/1i6oXabir-628csa2bQacHpc17Kmhec9EfkyHij9yQGM/edit#heading=h.vjzx5r62fvip) Here's the part that's probably relevant:

> 3.06 FEES
> The membership fee shall be an amount fixed from time to time by a majority vote of the Board, which vote shall become effective only when confirmed by a vote of the members at an annual or special general meeting.
>
> The Secretary shall notify the members (with the exception of honorary members) of the dues or fees at any time payable by them and, if any are not paid within 60 days of the date of such notice thereof, the members in default shall thereupon automatically cease to be members of the Society.
>
> Upon the failure of any member (with the exception of honorary members) to pay any membership fee, any subscription, or indebtedness due to the association, the Board may cause the name of such member to be removed from the register of members, but such member may be readmitted to membership by the Board upon such evidence as they may consider satisfactory.

I'd need to go back to our minutes for years past to find the motion, but I believe we voted on $20/year for a one year term. We also agreed at the last monthly meeting that members could pay us $20 via PayPal and we'd accept the fees PayPal charged us just to keep things simple.

File: pages/become_a_member.vue [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<template>
+    <div class="max-w-7xl mx-auto spx-6 px-6 lg:px-8">
+        <h1 class="text-2xl font-bold py-4">Become a Member</h1>
+        <p class="py-2">
+            Thank you for your interest in becoming a DES member! Firstly we would like to point out that <strong> you do not need to be a member to use the DES Slack, attend meetups and events, or to join the monthly society meeting and AGM. </strong> Membership is only required if you wish to vote, or use DES equipment and resources (like our YouTube channel or streaming tools.)
```
Comment (MarkBennett) [inline pages/become_a_member.vue:5]: So it seems to not want me to suggest changes to multiple lines, so hopefully this still makes sense...

```suggestion
            Thank you for your interest in becoming a DES member! Firstly we would like to point out that <strong> you do not need to be a member to use the DES Slack, attend meetups and events, or to join the monthly society meeting and AGM. </strong>
            
            Any member of the public is welcome to:

            * Join the Edmonton developer community            
            * Chat on our Slack
            * Attend DES supported meetups and events
            * Come out to DES monthly planning meetings
            * Participate in Working Groups
            
            <strong>Membership is only required if:</strong>
            
            * you wish to vote on society business
            * borrow DES equipment (cameras, mics, etc.)
            * use the DES resources (like our YouTube channel or streaming tools)
            * start an event or group officially supported by DES
            * use the DES name or branding
            * champion a Working Group
            * you want to give us $20/year to support the work we do (thank you!)
```

I'm thinking re-stating that you don't need to be a member in text and in bullets, along with a specific list of the times you do might help people understand the difference.

**Review by MarkBennett — CHANGES_REQUESTED** [review]
Looks like a great start @MandyMeindersma, thank-you!

I left a few suggestions on the content mostly just to try to drive home to people that you really only need to be a member if you want to run meetups or vote.

I was also wondering if you could change your editor to wrap lines at 80 characters? Maybe I'm blind but it seems like GitHub doesn't do that and it makes reading your changes hard.

Thanks! :)

File: pages/become_a_member.vue (hunk 2 above)
Comment (dboydgit) [reply pages/become_a_member.vue:8]: Right, I did say in my suggested wording that the fees are annual. I still don't think the original wording fits.

I also don't think that your statement (if you stop paying your fee then after a year you will no longer being a voting member) aligns with the bylaws. It's clear in the by-laws that membership isn't automatically revoked on non payment of fees, theres a process that needs to be followed to revoke membership (warning followed by 60 days grace period)

**Comment by MandyMeindersma** [comment]
Thank you all for the suggestions! I haven't forgot about this. I will probably just look at it on the weekend :)

**Comment by MarkBennett** [comment]
Hey @MandyMeindersma, no pressure here. Just wanted to see if there's anything you need on this PR to get it wrapped up?

**Comment by MarkBennett** [comment]
@allcontributors add @MandyMeindersma as code, design, and documentation

**Comment by NonMint** [comment]
@MandyMeindersma @MarkBennett what do we need to do to push this forward? This link would be valuable for meetings and meetups, have a consistent place to direct people too.

**Comment by MandyMeindersma** [comment]
@NonMint I have to actually go through and address the comments and read the bylaws for better wording 😮‍💨 

It's on my to-do list :P I swear I will get to it :P

**Comment by MandyMeindersma** [comment]
I am going to close this one and open a new one when I have time :P

**Pull request closed without merging**